benjaminkott/bootstrap_package
# 1326 hide breadcrumb divider element

# Pull Request

## Related Issues

* Fixes #1326

## Prerequisites

* [x] Changes have been tested on TYPO3 v12.5 LTS
* [x ] Changes have been tested on PHP 8.1

## Description

Override Bootstrap SCSS in order to prevent ::before element from beeing read in the screen reader. 

code infos here: https://a11ysupport.io/tests/tech__css__css_generated_content_alt

## Steps to Validate

1. build frontend
2. check if ::before element is using the news syntax "/" / ""
